state-synchronizer/src/chunk_request.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{Error, Result};
+use libra_types::crypto_proxies::LedgerInfoWithSignatures;
+use libra_types::transaction::Version;
+use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
+use std::fmt;
+
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq)]
+pub enum TargetType {
+    /// The response is built relative to the target (or end of epoch).
+    TargetLedgerInfo(LedgerInfoWithSignatures),
+    /// The response is built relative to the highest available LedgerInfo (or end of epoch).
+    /// The value specifies the timeout in ms to wait for an available response.
+    HighestAvailable { timeout_ms: u64 },
+    /// The response is built relative to a LedgerInfo at a given version.
+    Waypoint(Version),
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq)]
+pub struct GetChunkRequest {
+    /// The response should start with `known_version + 1`.
```

Comment:
you mean greater than `known_version` no?

Reply:
The client sends its `known_version`, the response starts with `known_version+1`.